packages/@okta/vuepress-site/copyright/index.md

```diff
@@ -0,0 +1,75 @@
+---
+
+# This file controls the content shown on the /copyright page.
+
+
+component: Copyright
+title: Copyright and Trademarks
+
+showBreadcrumb: False
+
+sections:
+
+- Copyright © 2022 Okta. All rights reserved.
+
+- Okta has made all efforts to ensure that this guide is accurate.
+
+- Okta Headquarters North America<br>
+  100 First Street<br>
+  San Francisco<br>
+  CA 94105<br>
+  USA
+
+- Okta®, Auth0®, and the Okta® and Auth0® Logos are registered trademarks of Okta, Inc. in the U.S. and other countries.
+
+- Okta's Terms of Service for its Intellectual Property <a href="https://www.okta.com/terms-of-service/" target="_blank">can be found here</a>.
```

Comment:
We avoid referencing links as "found here" or "here". Does this work instead?

```suggestion
- See <a href="https://www.okta.com/terms-of-service/" target="_blank">Okta's Terms of Service for its Intellectual Property</a>.
```

Reply:
Fine with me. Changed